fix c client in example, improve directions (#315)
KxSystems/docs: docs/kb/server-calling-client.md

# Server calling client

This demonstrates how to _simulate_ a [C client](../interfaces/c-client-for-q.md) handling a _get_ call from a kdb+ server. 

> The [Java interface](https://github.com/KxSystems/javakdb) allows the programmer to emulate a kdb+ server. The C interface does not provide a means to respond to a sync call from the server but async responses ([message type 0](../basics/ipc.md)) can be sent using [`k(-c,...)`](../interfaces/capiref.md#k-evaluate).

Reviewer: ```suggestion
The [Java interface](https://github.com/KxSystems/javakdb) allows you to emulate a kdb+ server. The C interface does not provide the ability to respond to a sync call from the server, however, async responses ([message type 0](../basics/ipc.md)) can be sent using [`k(-c,...)`](../interfaces/capiref.md#k-evaluate).
```
